web/src/user_group_edit.js

```diff
@@ -250,7 +250,8 @@ export function update_settings_pane(group) {
     $edit_container.find(".group-name").text(group.name);
     $edit_container.find(".group-description").text(group.description);
 
-    settings_org.discard_group_property_element_changes($("#id_can_mention_group"), group);
+    const $subsection = $edit_container.find(".settings-subsection-parent");
+    settings_org.discard_group_settings_subsection_changes($subsection, group);
```

Comment:
There is only one group setting currently hence calling the function to discard all elements in subsection is fine.
We can update this as a follow-up to only update the setting which is changed. Or we can do that when we add the next group setting.

Reply:
Do we need to add a code comment regarding that for the future?

Reply:
I have added a comment. Please have a look.